### ISSUE TITLE
latest_release v2 net plugin is wrong version

### ISSUE THREAD

**Issue by rade** [weaveworks/weave#3152]
```
$ docker plugin install weaveworks/net-plugin:latest_release
$ weave version
weave script <elided>
weave git-d0662b519f23
```
That should report as 2.0.5, since that is the latest release. Contrast with
```
$ docker plugin install weaveworks/net-plugin:2.0.5
$ weave version
weave script <elided>
weave 2.0.5
```
The [git revision](https://github.com/weaveworks/weave/commit/d0662b) mentioned above is for the 2.0.0 revision, but evidently prior to tagging.

**Comment by rade** [comment]
I looked around for a way to publish the 2.0.5 plugin under the latest_release tag. Alas there doesn't seem to be a way to tag plugins, other than creating them fresh with a different tag.

Plus there is the question why the discrepancy arose in the first place; afaict the release script is creating and publishing the plugin twice, with a version tag and the `latest_release` tag, so the reported version of both should be the same.

**Comment by rade** [comment]
Here's how I fixed this
```shell
plugin=weaveworks/net-plugin
vsn=2.0.5
docker plugin install $plugin:$vsn
docker plugin disable $plugin:$vsn
img=$(docker plugin inspect -f '{{.Id}}' $plugin:$vsn)
mkdir /tmp/net-plugin
sudo cp -ax /var/lib/docker/plugins/$img/rootfs /tmp/net-plugin
curl -o /tmp/net-plugin/config.json https://raw.githubusercontent.com/weaveworks/weave/$vsn/prog/net-plugin/config.json
sudo docker plugin create $plugin:latest_release /tmp/net-plugin
docker plugin push $plugin:latest_release
sudo rm -rf /tmp/net-plugin
```
and now...
```
$ docker plugin install weaveworks/net-plugin:latest_release
$ weave version
weave script <elided>
weave 2.0.5
```

@bboreham I am curious how this issue occurred, i.e. why was `latest_release` old and why was it not identifying itself as a versioned release? Seems to me that something must have broken during the release.

**Comment by bboreham** [comment]
Did this also fix #3150 ?

**Comment by rade** [comment]
> Did this also fix #3150 ?

No, that one is about the plugin in the official store. I have updated the title to make that clearer.

The change I made in #3156 makes that issue much less of a problem.

**Comment by bboreham** [comment]
For the 2.1.0 release I watched carefully, and all seemed to go fine.
`make plugin_publish` is the first thing the release script does in the `publish` phase.
